pyro/distributions/torch_distribution.py

```diff
@@ -225,6 +225,33 @@ class TorchDistribution(torch.distributions.Distribution, TorchDistributionMixin
     pass
 
 
+# TODO move this upstream to torch.distributions
```

Comment:
@alicanb Let's try to move this constraint upstream by next Pytorch release, and use it in the `Independent` distribution and all other multivariate distributions.